src/services/item/plugins/itemLike/index.ts

```diff
@@ -102,7 +102,7 @@ const plugin: FastifyPluginAsync = async (fastify) => {
           },
         };
         await actionService.postMany(member, buildRepositories(manager), request, [action]);
-        return newItemLike;
+        return newItemLike.id;
```

Comment:
Need to reflect in schema?

Reply:
In fact, the schema is already valid. The previous implementation was not respecting the schema